pymeasure/instruments/agilent/agilent8722ES.py

```diff
@@ -179,31 +192,32 @@ def frequencies(self):
         )
 
     @property
-    def data(self):
-        """ Returns the real and imaginary data from the last scan
+    def data_complex(self):
```

Comment:
This rename seems it would just arbitrarily break existing code using this. What is the benefit, as the docstring already made clear the purpose of `data`?